index_matroska.md

```diff
@@ -118,11 +118,14 @@ when, and only when, they appear in all capitals, as shown here.
 This document defines specific terms in order to define the format and application of `Matroska`.
 Specific terms are defined below:
 
-`Matroska`: a multimedia container format based on EBML (Extensible Binary Meta Language)
+`Matroska`:
+: A multimedia container format based on EBML (Extensible Binary Meta Language).
 
-`Matroska Reader`: A `Matroska Reader` is a data parser that interprets the semantics of a Matroska document and creates a way for programs to use `Matroska`.
+`Matroska Reader`:
+: A data parser that interprets the semantics of a Matroska document and creates a way for programs to use `Matroska`.
 
-`Matroska Player`: A `Matroska Player` is a `Matroska Reader` with a primary purpose of playing audiovisual files, including `Matroska` documents.
+`Matroska Player`:
+: A `Matroska Reader` with a primary purpose of playing audiovisual files, including `Matroska` documents.
```

Comment:
This sounds strange to me: a Matroska Reader… including Matroska documents. Is it not the primary purpose to play Matroska files and others are the “nice to have” part?

Reply:
Yes, it's a bit odd. You may raise a separate issue as this patch is a formatting patch (although it changes the text a bit to make it more logical with the formatting).

Reply:
The text reduction is similar to [this EBML patch](https://github.com/cellar-wg/ebml-specification/commit/06545b4e57e51db085370e2e5f1ee3c7fdde0e28).

Reply:
Yep, I’ll. I didn’t notice it in the old version.